changelog.md

```diff
@@ -1,5 +1,8 @@
 # Pelion E2E Python Test Library Changelog
 
+## 0.1.4  2019-12-xx
+- Automatic reset for the board
```

Comment:
Trivial point. Changelog entries should end with `.` to be complete sentences.

`Added automatic reset for the board.`